reports/lib-core.api.md

```diff
@@ -157,7 +157,7 @@ export type PluginLoaderOptions = Partial<{
     fetchImpl: ResourceFetch;
     fixedPluginDependencyResolutions: Record<string, string>;
     sharedScope: AnyObject;
-    postProcessManifest: (manifest: PluginManifest) => PluginManifest;
+    transformPluginManifest: TransformPluginManifest;
```

Comment:
@vojtechszocs looking good! FYI this is used by consoledot - https://github.com/RedHatInsights/insights-chrome/blob/master/src/components/RootApp/ScalprumRoot.tsx#L164

So please be sure to make this a major version bump when publishing.